### PR TITLE
Devi/pwm test tool

### PR DESCRIPTION
**Pull request by v-dmudili**
Created a folder for Pwm Test Tool app & added all the project files for windows 10 Iot Enterprise.

Following changes :

file modified : PwmTestTool.vcxproj
Updated the Target Platform Version from 10.0.17763 to 10.0.19041

file modified : README.md
Update Download link for Visual Studio 2019.
Update Download link for latest version of the Windows SDK.

### CODE REVIEW

File: BusTools/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline BusTools/.gitkeep]: Unneeded .gitkeep file. Remove?

File: BusTools/PwmTestTool/.gitkeep [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline BusTools/PwmTestTool/.gitkeep]: Unneeded .gitkeep file. Remove?

File: BusTools/readme.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- MinComm (UART)
+
+## Building
+
+1. Download [Visual Studio 2019](https://www.visualstudio.com/downloads/). Select options for C++, Windows UWP, and Windows Desktop app development. You may need to download the latest version of the [Windows SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk).
+1. From the start menu, run `Developer Command Prompt for VS 2019`
+1. Navigate to the root of this repository and run:
+
+```powershell
+msbuild /p:Platform=ARM /p:Configuration=Release
```
Comment (neilsh-msft) [inline BusTools/readme.md]: Update to Platform=x64?

File: BusTools/.gitkeep (hunk 1 above)
Comment (v-dmudili) [reply BusTools/.gitkeep]: removed...

**Review by neilsh-msft — APPROVED** [review]

**Pull request merged**